Tailwind Recipe fails with Blitz v 0.29.7

### What is the problem?
Getting error when trying to run `blitz install tailwind`: 

λ blitz install tailwind
You are using beta software - if you have any problems, please open an issue here:
      https://github.com/blitz-js/blitz/issues/new/choose

    Error: Cannot find module 'module-alias'

### Steps to Reproduce

1. Run  `blitz install tailwind` with a newly installed Blitz@0.29.7 on Windows

### Versions

λ blitz --version --verbose
You are using beta software - if you have any problems, please open an issue here:
      https://github.com/blitz-js/blitz/issues/new/choose

debug: blitzPkgPath: C:\git\kurs\dotjs-leaderboardd\node_modules\blitz\dist\index.js
debug: cliPkgPath: C:\git\kurs\dotjs-leaderboardd\node_modules\@blitzjs\cli\lib\src\index.js

Windows 10 | win32-x64 | Node: v12.20.1

blitz: 0.29.7 (global)
blitz: 0.29.7 (local)

  Package manager: yarn
  System:
    OS: Windows 10 10.0.19042
    CPU: (12) x64 Intel(R) Core(TM) i7-9750H CPU @ 2.60GHz
    Memory: 52.26 GB / 63.74 GB
  Binaries:
    Node: 12.20.1 - C:\Program Files\nodejs\node.EXE
    Yarn: 1.22.10 - ~\AppData\Roaming\npm\yarn.CMD
    npm: 6.13.6 - C:\Program Files\nodejs\npm.CMD
    Watchman: Not Found
  npmPackages:
    @prisma/cli: ~2.14 => 2.14.0
    @prisma/client: ~2.14 => 2.14.0
    blitz: 0.29.7 => 0.29.7
    react: 0.0.0-experimental-3310209d0 => 0.0.0-experimental-3310209d0
    react-dom: 0.0.0-experimental-3310209d0 => 0.0.0-experimental-3310209d0
    typescript: 4.1.3 => 4.1.3

### Other

Please include applicable logs and screenshots that show your problem.

--- a/packages/config/package.json
+++ b/packages/config/package.json
@@ -33,5 +33,8 @@
     "type": "git",
     "url": "https://github.com/blitz-js/blitz"
   },
-  "gitHead": "d3b9fce0bdd251c2b1890793b0aa1cd77c1c0922"
+  "gitHead": "d3b9fce0bdd251c2b1890793b0aa1cd77c1c0922",
+  "dependencies": {
+    "module-alias": "2.2.2"
+  }
 }

